tools/cli/package.json

```diff
@@ -7,6 +7,9 @@
     "engines": {
         "node": ">=14"
     },
+    "bin": {
+        "backtrace-js": "./lib/index.js"
```

Comment:
what other functions will this library have? So far we know only about the source map support. Do we plan to add anything else?

Reply:
I don't really know about that. We could certainly extend this to have other functionalities as the need for them arises.